tcf_website/templates/reviews/review.html

```diff
@@ -36,48 +40,69 @@
                           {% include "reviews/delete_confirm_modal.html" with review_id=review.id %}
                         </div>
                     </div>
-                    {% endif %}
-                    <p class="card-text" style="white-space: pre-line;">{{ review.text }}</p>
                 </div>
+                {% endif %}
 
-                <!-- Review Ratings -->
-                <div class="review-ratings d-flex justify-content-between align-items-end flex-wrap">
-                    <div class="d-flex">
-                        <div>
-                            <strong>
-                                <p id="review-average">
-                                    <i class="fa fa-star fa-fw"></i> {{ review.average|floatformat:2 }}
-                                </p>
-                            </strong>
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fas fa-thumbs-up"></i>
-                            {{ review.recommendability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-user"></i>
-                            {{ review.instructor_rating }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="far fa-smile-beam"></i>
-                            {{ review.enjoyability }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-dumbbell fa-fw"></i>
-                            {{ review.difficulty }}
-                        </div>
-                        <div class="ml-4 text-muted">
-                            <i class="fa fa-hourglass-half fa-fw"></i>
-                            {{ review.hours_per_week }}
+                <div class="review-content">
+                  <p class="card-text mb-2" style="white-space: pre-line;">{{ review.text }}</p>
+                </div>
+
+                <div class="review-footer row mt-3">
+                    <!-- Review Ratings -->
+                    <div class="review-ratings col-sm-8 d-flex align-items-end">
+                        <div class="d-flex text-center flex-wrap mb-2">
```

Comment:
Do you think these two containers (`review-ratings col-sm...` and `d-flex text-center...`) could be combined? It seems a bit  redundant and they both have flex features